libr/main/rabin2.c

```diff
@@ -1173,7 +1173,7 @@ R_API int r_main_rabin2(int argc, char **argv) {
 		rabin_do_operation (bin, op, rad, output, file);
 	}
 	if (isradjson) {
-		r_cons_print ("}");
+		r_cons_print ("}\n");
```

Comment:
we should use pj

Reply:
yes, but that is out of the scope of this pr.